core/src/client/mod.rs

```diff
@@ -1,9 +1,10 @@
 use eyre::Result;
 use jsonrpsee::types::error::CallError;
+use std::convert::From;
 
 use reth_primitives::{
-    rpc::{BlockId, BlockNumber, Bytes, Log, H160, H256},
-    Address, H256 as PrimitiveH256, U256, U64,
+    rpc::{BlockId, BlockNumber, Bytes as RpcBytes, Log, H160 as RpcH160, H256},
```

Comment:
Type conundrum intensifies..
This pains me, what is the status of the reth types problem?